src/hivpy/demographics.py

```diff
@@ -6,6 +6,7 @@
 import pandas as pd
 from scipy.interpolate import interp1d
 
+import hivpy.column_names as col
```

Comment:
I wonder if it's preferable to import specific names directly (`from hivpy.column_names import AGE_GROUP, SEX`), to:
- save some typing
- see at a glance which columns are referenced in a particular file

I don't have strong feelings either way though.